ledger-tool/src/main.rs

```diff
@@ -2571,8 +2571,8 @@ fn main() {
                     };
                     process_options.halt_at_slot = Some(parent_slot);
 
-                    // PrimaryForMaintenance needed over Secondary to purge any
-                    // existing simulated shreds from previous runs
+                    // PrimaryForMaintenance needed to purge(write) any existing simulated shreds
```

Comment:
Sorry, this was a typo in my comment 🤦 ; can you change `purge(write)` to `purge (write)`